Src/Base/AMReX_GpuFuse.cpp

```diff
@@ -0,0 +1,241 @@
+#include <AMReX_Gpu.H>
+#include <AMReX_ParmParse.H>
+#include <AMReX_BLProfiler.H>
+#include <AMReX_OpenMP.H>
+
+namespace amrex {
+namespace Gpu {
+
+#ifdef AMREX_USE_CUDA
+
+namespace {
+    bool s_in_fuse_region = false;
+    // Fusing kernels with elements greater than this are not recommended based tests on v100
```

Comment:
Does inFuseRegion do anything yet? Searching s_in_fuse_region and inFuseRegion, as well as checking Register and Launch, I'm not seeing where it's used.

Also, with the explicit writing style shown in the example, do we want the default to be off?

Reply:
Not yet.  I have a branch using it waiting on this PR.